proto/spacemesh/types.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+package spacemesh;
+
+message SimpleInt {
```

Comment:
Do we still need this if we have Amount, below?

Reply:
Yes - for methods int params which are not coin amounts.

Reply:
Strictly speaking, according to the guidelines I read for API design, each RPC method should have its own unique parameters. They shouldn't be shared, to avoid issues, so there should be no "generic" types. Anyway I think it's okay for now.